### PR TITLE
Add prototypes and data types for sceClibMspace* functions.

### PR DESCRIPTION
**Pull request by bythos14**

### CODE REVIEW

File: include/psp2/kernel/clib.h [hunk 1]
```diff
@@ -14,6 +14,21 @@
...
+ */
+typedef void* SceClibMspace;
+
+/**
+ * Structure for stats of SceClibMspace
+ */
+typedef struct SceClibMspaceStats {
+    SceSize capacity;     //!< Capacity of the Mspace
+    SceSize unk;          //!< Unknown, value is equal to capacity
+    SceSize peakInUse;    //!< Peak memory allocated
```
Comment (d3m3vilurr) [inline include/psp2/kernel/clib.h:28]: can you use `snake_case` for the field name?
```suggestion
    SceSize peak_in_use;    //!< Peak memory allocated
```

File: include/psp2/kernel/clib.h [hunk 2]
```diff
@@ -14,6 +14,21 @@
...
+    SceSize currentInUse; //!< Current memory allocated
```
Comment (d3m3vilurr) [inline include/psp2/kernel/clib.h:29]:
```suggestion
    SceSize current_In_use; //!< Current memory allocated
```

File: include/psp2/kernel/clib.h [hunk 3]
```diff
@@ -51,6 +66,22 @@ int sceClibMemcmp(const void *s1, const void *s2, SceSize len);
 
 void *sceClibMemchr(const void *src, int ch, SceSize len);
 
+SceClibMspace sceClibMspaceCreate(void* memBlockAddr, SceSize size);
```
Comment (d3m3vilurr) [inline include/psp2/kernel/clib.h:69]:
```suggestion
SceClibMspace sceClibMspaceCreate(void* memblock_addr, SceSize size);
```
or
```suggestion
SceClibMspace sceClibMspaceCreate(void* mem_block_addr, SceSize size);
```

but how about just `memblock`?

**Review by d3m3vilurr — CHANGES_REQUESTED** [review]

File: include/psp2/kernel/clib.h (hunk 3 above)
Comment (bythos14) [reply include/psp2/kernel/clib.h]: Requested adjustments have been made.

**Comment by d3m3vilurr** [comment]
LGTM now. thanks for your contribution!

**Pull request merged**